hometask6

```diff
@@ -0,0 +1,68 @@
+# Task 1
```

Comment:
Uncomment your code, please

Reply:
All, or just the first task?

Reply:
All. You shouldn't send for review commented code